spec/db_methods_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require_relative '../lib/new_super_codebreaker_2021'
+require_relative '../lib/user'
+
+RSpec.describe NewSuperCodebreaker2021::Game do
+  let(:game) { described_class.new }
+  let(:user) { User.new('Name', 0) }
```

Comment:
Can we use [`Faker`](https://rubygems.org/gems/faker) or [`FFaker`](https://rubygems.org/gems/ffaker) for a test values?

Reply:
added